rotkehlchen/api/rest.py

```diff
@@ -3982,18 +3984,17 @@ def get_user_added_assets(self, path: Optional[Path]) -> Response:
             status_code=HTTPStatus.OK,
         )
 
-    @require_loggedin_user()
-    def import_user_assets(self, path: Path) -> Response:
+    def _import_user_assets_file(self, file_path: Path) -> Optional[Response]:
```

Comment:
```suggestion
    def _import_user_assets_file_or_error(self, file_path: Path) -> Optional[Response]:
    """Import user assets file or return an error response in case something fails"""
```

But I don't like this approach, you needlessly use exceptions here to control flow.